distribution/docs/src/main/resources/_contents/_plugins/registry-policy-plugin-contents.adoc

```diff
@@ -0,0 +1,17 @@
+
+==== Registry Policy Plugin
+
+{description}
+
+===== Installing Registry Policy Plugin
+
+The Registry Policy Plugin is is not installed by default with a standard installation in the ${app-name} application.
+
+===== Configuring Registry Policy Plugin
+
+{configuration}
```

Comment:
. Navigate to the ${admin-console}.
. Select the *${ddf-registry}* application tile.
. Select the *Configuration* tab.
. Select the *Registry Policy Plugin* configuration.
 +
 +include::{adoc-include}/_tables/ ... policyplugin adoc link here